pkg/notification/notice.go

```diff
@@ -29,17 +33,18 @@ type VersionChecker struct {
 }
 
 // NewVersionChecker creates a new VersionChecker with the default
-// updates API URL. The URL can be overridden by passing an Option
-// to the NewVersionChecker function.
-func NewVersionChecker(opts ...Option) *VersionChecker {
+// updates API URL.
+func NewVersionChecker(commandName string, cliOptions *flag.Options) *VersionChecker {
 	v := &VersionChecker{
-		updatesApi:     "https://check.trivy.dev/updates",
-		currentVersion: app.Version(),
+		updatesApi:       "https://check.trivy.dev/updates",
+		currentVersion:   app.Version(),
+		skipVersionCheck: cliOptions.SkipVersionCheck,
+		quiet:            cliOptions.Quiet,
+		disableTelemetry: cliOptions.DisableTelemetry,
+		commandName:      commandName,
+		cliOptions:       cliOptions,
```

Comment:
You added duplicates here:
you can take skipVersionCheck, quiet and disableTelemetry from cliOptions